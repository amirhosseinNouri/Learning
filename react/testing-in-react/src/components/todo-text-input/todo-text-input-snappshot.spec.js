import { expect } from '@jest/globals';
import React from 'react';
import renderer from 'react-test-renderer';
import TodoTextInput from './todo-text-input';

test('Snapshot test', () => {
  const noop = () => {};
  const component = renderer.create(<TodoTextInput onSave={noop} editing />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
