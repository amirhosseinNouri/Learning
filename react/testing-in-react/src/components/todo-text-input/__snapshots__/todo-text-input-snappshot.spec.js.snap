// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test 1`] = `
<input
  autoFocus={true}
  className="edit"
  onBlur={[Function]}
  onChange={[Function]}
  onKeyDown={[Function]}
  type="text"
  value=""
/>
`;
